### DISCUSSION TITLE
Error when renaming filename through Templater

### DISCUSSION THREAD

**Discussion by yinlokc** [SilentVoid13/Templater#668] [Help] [answered]
I would like to input the file name from the prompt and rename the file by using Templater.

This is what I put 

<%* title = tp.system.prompt ("Please enter a value") %>
<%* tp.file.rename(title+'a') %>

and result of the file name is "[object Promise]a"

Do anyone know why?

**Comment by AB1908** [comment, marked as answer]
You need to `await` both the prompt and the rename.

**Reply by yinlokc** [reply]
Thanks very much!
